UXWWhoWeAre.js

```diff
@@ -49,6 +125,26 @@ function UXWWhoWeAre(props) {
           <li>Initiative - We wouldn’t be here without the volunteers that step up and get things done.</li>
         </ul>
       </UXWSubsection>
+      <UXWSubsection anchor="about--wizards" title="Contributors">
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <List>
+      {data.map(({image, name, role, description, link}) => (
+      <div className="row">
+        <div className="column" style={columnStyle}>
+          <div className="card" style={cardStyle}>
+            <img src={image} alt="Jane" style={{width: '400px', height: '300px'}} />

```

Comment:
Alt should probably be the name. Also might be easier to not try to force the image size like this, and rather just crop the image properly. That way you can let it scale proportionally. Should we all crop our images to be 400px width and 300px height?

Reply:
fixed alt to be name. will work on the images and cropping